clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -694,6 +694,16 @@ void tools::addLTOOptions(const ToolChain &ToolChain, const ArgList &Args,
     CmdArgs.push_back(Args.MakeArgString(Twine(PluginOptPrefix) +
                                          ParallelismOpt + Parallelism));
 
+  // Pass down GlobalISel options.
+  if (Arg *A = Args.getLastArg(options::OPT_fglobal_isel,
+                               options::OPT_fno_global_isel)) {
+    // Parsing -fno-global-isel explicitly gives architectures that enable GISel
+    // by default (e.g. AArch64) a chance to disable it.
```

Comment:
"e.g. AArch64" in "architectures that enable GISel by default (e.g. AArch64)" is not accurate. It's the default for -O0 and not for some code models. Perhaps just omit the "e.g." part.

Reply:
Done